demos/sofaboot/README.md

```diff
@@ -3,11 +3,26 @@
 This project demonstrates how to execute an unmodified sofaboot projects with Occlum.
 
 1. Download and build sofaboot project
+
+* Use openjdk 8
+```
+./download_compile_sofaboot.sh jdk8
+```
+
+* Use openjdk 11
 ```
 ./download_compile_sofaboot.sh
 ```
 
+
 2. Run `sofaboot sample standard web` on Occlum
+
+* Run with openjdk 8
+```
+./run_sofaboot_on_occlum_jdk8.sh
```

Comment:
Maybe we should combine the script into ./run_sofaboot_on_occlum.sh. The use could run command "./run_sofaboot_on_occlum.sh jdk8" to trigger the demo.